src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -835,9 +835,19 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
-                        ->booleanNode('disable_not_compromised_password')
-                            ->defaultFalse()
-                            ->info('Disable NotCompromisedPassword Validator: the value will always be valid.')
+                        ->arrayNode('not_compromised_password')
+                            ->addDefaultsIfNotSet()
+                            ->info('Non-compromised password validator configuration')
+                            ->children()
+                                ->booleanNode('enable')
```

Comment:
name it `enabled` to be consistent with other parts of the config enabling or disabling things

Reply:
Done. Maybe it would even make more sense to change the naming in the constraint as well (i.e. change everything from disabled to enabled)?